tsdb/head_bench_test.go

```diff
@@ -79,6 +86,70 @@ func BenchmarkHeadStripeSeriesCreate_PreCreationFailure(b *testing.B) {
 	}
 }
 
+func BenchmarkHead_WalCommit(b *testing.B) {
+	seriesCounts := []int{100, 1000, 10000}
+	series := genSeries(10000, 10, 0, 0)
+	histograms := genHistogramSeries(10000, 10, 0, 119, 1, true)
```

Comment:
Most of real life histograms have  integer bucket counts, so `true`->`false`.
Also to not conflict with the float series, let's have 11 labels.
```suggestion
	histograms := genHistogramSeries(10000, 11, 0, 119, 1, false)
```